app/src/main/res/values/strings.xml

```diff
@@ -28,5 +28,6 @@
     <string name="web_desktop_extension">Web/Desktop Extension</string>
     <string name="notifications">Notifications</string>
     <string name="emergency_contact">Emergency Contact</string>
-    <string name="megan_fox">Megan Fox</string>
+    <string name="okay_text">OK</string>
+    <string name="megan_fox">Megan Fox</string>zz
```

Comment:
Why do we have a Mega Fox with "zz" at the end?

Reply:
I have no clue why the Megan Fox is there, I'll get rid of the 'zz' 😁